Parent/Parent/Thresholds/StudentSettingsViewController.swift

```diff
@@ -102,7 +102,7 @@ open class StudentSettingsViewController : FormViewController {
         let sectionTitle = NSLocalizedString("Alert me when:", comment: "Alert Section Header")
         form +++
             Section() {
-                $0.header = studentSectionHeaderView()
+                $0.header = self.studentSectionHeaderView()
```

Comment:
should this be weak or should no worries here?

Reply:
I think we are good. `Section` does not hold a reference to the `initializer` block.

This is what it looks like:

```swift
    public init(_ initializer: @escaping (Section) -> Void) {
        initializer(self)
    }
```

Not really sure why they mark is as `@escaping`.

Reply:
Also, we don't hold a reference to the Section. Looks like it's just used to build a view then de-initialize.